Simple_Prep.sh

```diff
@@ -3,7 +3,13 @@
 set -e
 
 # Check if FSL is available
-[ ! $(command -v bet) ] && echo "No bet: Make sure an FSL version is installed and available." && exit 1
+if [ ! $(command -v bet) ]; then
```

Comment:
cool.  didn't know about `command -v`... usually used `which` but that one is not fully portable iirc, so for those cases used 'hash' ;-)